chef_master/source/server_security.rst

```diff
@@ -226,14 +226,22 @@ Private Certificate Authority
 -----------------------------------------------------
 If an organization is using an internal certificate authority, then the root certificate will not appear in any ``cacerts.pem`` file that ships by default with operating systems and web browsers. Because of this, no currently deployed system will be able to verify certificates that are issued in this manner. To allow other systems to trust certificates from an internal certificate authority, this root certificate will need to be configured so that other systems can follow the chain of authority back to the root certificate. (An intermediate certificate is not enough because the root certificate is not already globally known.)
 
-To use an internal certificate authority, append both the server and root certificates into a single ``.crt`` file. For example:
+To use an internal certificate authority, append both the server (optionally, any intermediate cert as well) and root certificates into a single ``.crt`` file. For example:
 
 .. code-block:: bash
 
-   $ cat server.crt root.crt >> /var/opt/opscode/nginx/ca/FQDN.crt
+   $ cat server.crt [intermediate.crt] root.crt >> /var/opt/opscode/nginx/ca/FQDN.crt
+
+You can check the validity of your new combined cert like this, on the Chef Server. The cacert.pem only needs to contain your root CA's cert file. This is not the usual treatment, but mimics how ChefDK behaves after a ``knife ssl fetch`` followed by a ``knife ssl verify``
```

Comment:
@sean-horn Thank you for writing this up! Can you clarify which code block (line 233 or line 237) you're referring to when you say "like this" in line 235?

Reply:
@mjingle I put the discussion before the code blocks here, so line 235 is referring to line 237.